rover-apps/science/include/AdafruitSTEMMAModule.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "Module.h"
+#include "mbed.h"
+#include "AdafruitSTEMMAModule.h"
+
+class AdafruitSTEMMAModule final : public Module {
+    public:
+        AdafruitSTEMMAModule();
+
+        void periodic_1s(void) override;
+        void periodic_10s(void) override {}
+        void periodic_100ms(void) override {}
+        void periodic_10ms(void) override {}
+        void periodic_1ms(void) override {}
+
+        AdafruitSTEMMA moistureSensor;
```

Comment:
Not sure if this is the right approach, to declare a sensor in the module header, but don't know where else it should be declared

Reply:
Also forgot to initialize it with the I2C pins, will fix that up

Reply:
> Not sure if this is the right approach, to declare a sensor in the module header, but don't know where else it should be declared

Yep that's fine, please make it a private member though